python/py-llfuse/Portfile

```diff
@@ -5,11 +5,11 @@ PortGroup           python 1.0
 PortGroup           fuse 1.0
 
 name                py-llfuse
-version             1.3.8
+version             1.5.0
 revision            1
```

Comment:
the "revision" field should be reset to "0" when updating the "version"

Reply:
when you nake that change also please add the following line to the body of the commit message:

Closes: https://trac.macports.org/ticket/69071

Reply:
Thanks. That's all done 👍